### ISSUE TITLE
issues when I run flutter doctor after set $PATH

### ISSUE THREAD

**Issue by chudasaarjun** [flutter/flutter#15775]
Error is :
FlutterProjects/flutter/bin/flutter: line 33: sed: command not found

**Comment by DanTup** [comment]
What platform are you on? If Windows, does it work if you run `flutter.bat doctor` instead of `flutter doctor`?

If you're on Mac or Linux, which distribution/version and what happens if you run `which sed`?

**Comment by chudasaarjun** [comment]
I am using Mac OS High Sierra , version 10.13.1

**Comment by DanTup** [comment]
What happens if you run `which sed` in the terminal where this happens?

**Comment by DanTup** [comment]
Maybe same issue as #15927?

**Comment by iamSahdeep** [comment]
Yes, @chudasaarjun  you can try changing Double Quotation to Single in line 33 of flutter script.

**Comment by Mahi-K** [comment]
hi @chudasaarjun I think may be your path variables are wrong, please reset all the variables and this should work fine. I had the same problem and recreated all the paths and the commands seems to be working fine.

**Comment by MahMoos** [comment]
like @Mahi-K said, you must have something wrong with your path variables. I had the same stupid issue. 
Your .bath_profile might look like this:
`export PATH=[/Users/username/Development]/flutter/bin:$PATH`
which is wrong and should be:
`export PATH=/Users/username/Development/flutter/bin:$PATH`

**Comment by zoechi** [comment]
Looks like a support request that was solved. Thanks @MahMoos 

Closing. Feel free to add a comment if you are still seeing this to have it reopened.
